src/packages/auth/functions.ts

```diff
@@ -115,7 +117,7 @@ export async function login(username: string, password: string): Promise<LoginRe
     return {
       message: 'Authentication is disabled',
       success: true,
-      token: generateJwt('unknown', '', 'admin', ['admin']),
+      token: generateJwt(username || 'unknown', '', 'admin', ['admin', 'user']),
```

Comment:
Why add user here?

Reply:
To allow for the case when authentication is disabled, the user is still able to swap roles if they want to.